src/props.supports.css

```diff
@@ -8,9 +8,16 @@
 :where(html) {
   --isLTR: 1;
   --isRTL: -1;
-  
+
   &:dir(rtl) {
+      /* PostCSS compiles this to [dir="rtl"] :where(html) for older browsers, which doesn't work */
```

Comment:
adding `'dir-pseudo-class': false,` to https://github.com/argyleink/open-props/blob/main/postcss.config.cjs#L17 will resolve this. postcss is attempting to polyfill it and we want just nesting.